Documentation/README_Interactable.md

```diff
@@ -0,0 +1,285 @@
+# Interactable
+![Interactable](/External/ReadMeImages/Interactable/InteractableExamples.png)
+
+## How to use Interactable
+Add the Interactable Component to a GameObject
+
+![Interactable](/External/ReadMeImages/Interactable/InteractableInspector_basicSteps.png)
+
+1. A Collider must exist on the GameObject with the Interactable or the child of the Interactable for it to receive input.
+2. Use the OnClick event to make something happen.
+3. Add visual feedback by linking a Target to a Profile and assigning a Theme.
+
+
+## Input Settings
+The basic features allow for button style interactions, such as pointer focus and clicks, that maps to interaction states to drive themes which are setup through the Interactable profile. Controller or hand focus, down, up and click (both near and far) are handled. Functionality can be extended using external scripts that can set state manually.
+
+<img src="/External/ReadMeImages/Interactable/InputFeatures_short.png" width="450">
+
+### Input Actions
+Select the action, from the input configuration or controller mapping profile, that the Interactable should react to.
+See the InputProfile or the DefaultMixedRealityInputActionsProfile for more on how Input Actions are setup and intended to be used in the application.
+ 
+### Enabled
+Sets the Interactable's enabled state, which will disable some input handling and update the themes to reflect the current state which is disabled.
+This is different from disabling input all together (using Enable Input) This means we have a button that would normally be interactive, but at this moment it is disabled (could be like a submit button waiting for all the required fields to be completed) and has a visual look and feel to denote it's disabled state.
+ 
+### IsGlobal
+Focus is not required to detect input actions, default behavior is false.
+ 
+### Voice Commands
+A voice command to trigger an OnClick event. This will also trigger a quick state change to drive any themes visuals.
+ 
+Note: Make sure there is a unique voice command on each button. The Voice Recognizer is global (even if the Interactable isn't) and will not register the same voice command twice; in this case an error will be thrown.
```

Comment:
Do you know if this is a runtime error, or in-editor error? i.e. will I only find out about this problem when things crash on the device, or will I get a chance to know in the editor.

Reply:
I have seen this error in the Editor.